spec/unit/work_spec.rb

```diff
@@ -207,182 +219,186 @@ def run
   end
 
   describe "when encountering an error" do
-    it "should exponentially back off the job" do
-      ErrorJob.enqueue
+    describe "when using the default strategy" do
```

Comment:
It may still be worth adding a spec which tests the delegation in the case where a job overrides the default strategy. That delegation is becoming part of the Que API.